web/ui/src/common/baseFactory.js

```diff
@@ -65,6 +65,9 @@
                     }
 
                     deferred.resolve();
+                })
+                .error((data, status) => {
+                    console.error("Unable to update factory", data);
```

Comment:
Good point. As discussed, we could try exponential backup after n errors, and display a notification in the UI that there is a problem with the endpoint. This probably should go into a separate PR/ticket